src/main/java/com/controller/MailController.java

```diff
@@ -0,0 +1,37 @@
+package com.controller;
+
+import com.dto.MailDto;
+import com.model.Mail;
+import com.service.MailService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+public class MailController {
+
+    MailService mailService;
+
+    @GetMapping("/mails")
+    public List<Mail> getMails(){
+        return mailService.getAll();
+    }
+
+    @PostMapping()
+    public void addMail(@RequestParam String sender,@RequestParam String receiver,
```

Comment:
You should generally use request body to transfer data from client to server due to a number of reasons:
1. URL length is limited
2. You may be passing sensitive data which, unless encrypted in request body, will be exposed over the network, which is obviously a bad thing